CHANGELOG.md

```diff
@@ -8,12 +8,26 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ### Added
+- Added button to make a new query
```

Comment:
Wouldn't write this. Rather document in #80 that in addition, you added a button to change variables.